Mathlib/Algebra/Order/LatticeGroup.lean

```diff
@@ -276,7 +276,7 @@ theorem mul_inf_eq_mul_inf_mul [CovariantClass α α (· * ·) (· ≤ ·)] (a b
     c * (a ⊓ b) = c * a ⊓ c * b := by
   refine' le_antisymm _ _
   rw [le_inf_iff, mul_le_mul_iff_left, mul_le_mul_iff_left]
-  simp
+  simp only [ge_iff_le, inf_le_left, inf_le_right, and_self]
```

Comment:
I note that `simp?` has a bug where it often redundantly includes `ge_iff_le`. Can it be removed here (and in two other spots below)?
```suggestion
  simp only [inf_le_left, inf_le_right, and_self]
```

Reply:
Done!